Core/DISET/AuthManager.py

```diff
@@ -256,6 +262,25 @@ def getUsername(self, credDict):
       return True
     return False
 
+  def isUserSuspended(self, credDict):
+    """ Discover if the use is in Suspended status
+
+    :param dict credDict: Credentials to check
+    :return: Boolean True if user is Suspended
+    """
+    if self.KW_USERNAME not in credDict:
```

Comment:
You do that just before calling this method, so do you need to call aget `getUsername` ?

Reply:
Well, I can presume that the method can be called in some other context as well (although not the case currently), so, the precaution